crates/cxx-qt-build/src/dir.rs

```diff
@@ -121,6 +121,10 @@ pub(crate) fn header_root() -> PathBuf {
     crate_target().join("include")
 }
 
+pub(crate) fn header_root_interface() -> PathBuf {
+    target().join("crates-include")
```

Comment:
probably want a better name like `export-include` and sort out various naming across the codebase